assets/javascripts/wizard/routes/custom.js.es6

```diff
@@ -12,6 +13,47 @@ export default Ember.Route.extend({
     return findCustomWizard(params.wizard_id, this.get("queryParams"));
   },
 
+  renderTemplate() {
+    this.render("custom");
+    const wizardModel = this.modelFor("custom");
+    const stepModel = this.modelFor("custom.step");
+
+    if (wizardModel.get("first_step.id") !== stepModel.id) {
```

Comment:
@fzngagan This needs to be wrapped in a wizard setting. Not all users will want this feature.